docs/index.rst

```diff
@@ -0,0 +1,106 @@
+Shotgun Hiero Export API reference, |release|
+###################################################
+
+
+The ``tk-hiero-export`` app adds custom Shotgun export processors to the Hiero/Nuke Studio export framework.
+
+Via custom export processors, the user can create and update entities in the current Project in Shotgun.
+
+During the export process a number things can happen:
+
+- Status of associated Shot entities can be updated
+- Nuke scripts can be written into the project's filesystem structure for each Shot that's processed
+- Sequence and Shot entity data can be updated in Shotgun
+- Cuts can be update to include new CutItems built from the exported sequences
+
```

Comment:
Is there good hiero API documentation? Perhaps it's worth including a link here in the intro?